Source/Core/DolphinQt/Settings/GeneralPane.cpp

```diff
@@ -83,7 +85,21 @@ void GeneralPane::OnEmulationStateChanged(Core::State state)
   const bool running = state != Core::State::Uninitialized;
 
   m_checkbox_dualcore->setEnabled(!running);
+#ifdef USE_RETRO_ACHIEVEMENTS
+  bool hardcore = Config::Get(Config::RA_HARDCORE_ENABLED);
+  m_checkbox_cheats->setEnabled(!running && !hardcore);
```

Comment:
This makes sense but it's a little unfortunate.  We already have this concept of `CheatWarningWidget` being used when cheats are disabled.  Feels like users might expect this warning to be there instead of disabling the cheat enable flag instead.  So I'd almost suggest adding `HardcoreWarningWidget` in the places where `CheatWarningWidget` is currently used, to have a bit more consistency.

What do you think?

Reply:
¿Por qué no los dos?

Reply:
I see what you're saying :)

I guess I had a couple thoughts:

1)  In general, I was trying to get away from the tooltip as being used to show that hardcode mode was on.  It's not something that will be obvious to the user to hover over to see why their control is disabled.
2) I was trying to avoid adding a tool tip to this section of the Dolphin settings.  While we eventually intend to add tooltips to all settings, having just _one_ in this pane seems really out of place.  Again, a user won't think to look there.

Reply:
Ohhhh I get you ... dang, I forgot I had to refactor this into a tooltip. Okay, I'll revert that.